cluster/get-kube.sh

```diff
@@ -162,8 +178,20 @@ case "${machine}" in
 esac
 
 file=kubernetes.tar.gz
-release=${KUBERNETES_RELEASE:-$(get_latest_version_number)}
-release_url="${KUBERNETES_RELEASE_URL}/${release}/${file}"
+release=${KUBERNETES_RELEASE:-"release/stable"}
+
+# Validate Kubernetes release version.
+# Translate a published version <bucket>/<version> (e.g. "release/stable") to version number.
+set_binary_version "${release}"
+if [[ ${KUBE_VERSION} =~ ${KUBE_RELEASE_VERSION_REGEX} ]]; then
+  release_url="${KUBERNETES_RELEASE_URL}/${KUBE_VERSION}/${file}"
+elif [[ ${KUBE_VERSION} =~ ${KUBE_CI_VERSION_REGEX} ]]; then
```

Comment:
We [call this script](https://github.com/kubernetes/test-infra/blob/5702a8cfeccd6df74ba0d2a05797a2735784bd67/jenkins/e2e-image/e2e-runner.sh#L121-L145) from CI, [overriding `KUBERNETES_RELEASE_URL`](https://github.com/kubernetes/test-infra/blob/5702a8cfeccd6df74ba0d2a05797a2735784bd67/jenkins/e2e-image/e2e-runner.sh#L91) in some cases to point to a different path. I'm worried that this might break that pattern.